vite.config.mjs

```diff
@@ -0,0 +1,37 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import svgr from 'vite-plugin-svgr'
+
+export default defineConfig({
+  plugins: [
+    react({
+      jsxRuntime: 'classic'
+    }),
+    svgr()
+  ],
+  esbuild: {
+    // We use function names for generating readable error messages, so we want
+    // them to be preserved when building and minifying.
+    keepNames: true
+  },
+  build: {
+    minify: false,
+    cssCodeSplit: true,
+    lib: {
+      entry: 'src/export.js',
+      fileName: format => `${format}/index.js`,
+      name: 'federationInfo',
+      formats: ['umd']
```

Comment:
we should export `esm`  and `cjs` as well

Reply:
Ah I thought we don't need them since we only referencing umd on the server.

Will try to add them

Reply:
`esm` and `cjs` builds with vite are splitting css into a separate file by default, and from the looks of it would need a separate plugin to handle that. 

If we don't need cjs and esm right now I would  suggest we skip it until we need it?

Reply:
Added `esm` and `cjs` builds, they have the separate css for now.